src/internal/connector/sharepoint/queries.go

```diff
@@ -0,0 +1,21 @@
+package sharepoint
```

Comment:
I like the idea of consolidating queries. Thoughts on defining a SharePoint query interface and stashing that in Graph Connector? We could use that to start mocking these queries.

Reply:
I'll look into it, sure.